### PR TITLE
Fix English genisys.yml

### PR DESCRIPTION
**Pull request by ABAxStorm**
### Description

This corrects some grammar and spelling for the genisys.yml comments in the file.

### Reason to modify

To fix grammar and spelling mistakes in the genisys.yml file
### Tests & Reviews

This does not alter any actual code, just the comments.

### CODE REVIEW

File: src/pocketmine/resources/genisys_eng.yml [hunk 1]
```diff
@@ -105,23 +105,23 @@ synapse:
...
+ #The max players' on the MOTD
  motd-max-players: 0
- #The max players' number of query，0=Default
+ #The max players' on the query. 0=Default
  query-max-players: 0
- #Show the number of all players on motd
+ #Show the number of all players on MOTD
  motd-all-players: false
- #Show the number of all players on Query
+ #Show the number of all players on query
```
Comment (ishitatsuyuki) [inline src/pocketmine/resources/genisys_eng.yml]: Query is a protocol, so it should be captitalized.

File: src/pocketmine/resources/genisys_eng.yml [hunk 2]
```diff
@@ -105,23 +105,23 @@ synapse:
...
  query-all-players: false
- #Show the number of online players on motd
+ #Show the number of online players on MOTD
  motd-players: false
- #Show the number of online players on Query
+ #Show the number of online players on query
```
Comment (ishitatsuyuki) [inline src/pocketmine/resources/genisys_eng.yml]: Just a reminder, this too.

File: src/pocketmine/resources/genisys_eng.yml [hunk 3]
```diff
@@ -105,23 +105,23 @@ synapse:
...
+ #Update Query automatically
  query-auto-update: false
- #Query update periodical
+ #Update Query periodically
  query-tick-update: true
- #The max players' number of Motd
+ #The max players' on the MOTD
  motd-max-players: 0
- #The max players' number of query，0=Default
+ #The max players' on the query. 0=Default
```
Comment (ishitatsuyuki) [inline src/pocketmine/resources/genisys_eng.yml]: And also this one. (see below.)

**Comment by ABAxStorm** [comment]
I'll be able to fix it soon when my computer is back up

**Comment by ABAxStorm** [comment]
I believe I rebased, let me know if there's any problems.

**Pull request merged**